### PR TITLE
Add project files

### PR DESCRIPTION
**Pull request by abulyousuf**
Please, review my html and css files for 'Four card feature section' project. Thanks in advance.

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+html, body {
+    font-family: 'Poppins', sans-serif;
+    font-weight: 200;
+    font-size: 15px;
+    background-color: hsl(0, 0%, 98%);
```
Comment (asjoss) [inline css/styles.css:11]: Hi Abul :) Your code is great so im just asking questions about things i'm not aware of; What does the hs1 mean (& the %'s)?

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+footer a {
+    text-decoration: none;
+}
+
+@media (min-width: 1000px) {
+    main {
+        margin: 30px 0;
+    }
+
```
Comment (asjoss) [inline css/styles.css:90]: I like how you set the .main and .container to those widths as that keeps the cards in the middle/centre of the viewport 🙌🙌

File: css/styles.css (hunk 1 above)
Comment (abulyousuf) [reply css/styles.css:11]: Hi Aaron, thank you. It's color values like RGB, hexadecimal.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Four card feature section</title>
+    <link rel="shortcut icon" type="image/png" href="./images/favicon-32x32.png">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
```
Comment (asjoss) [inline index.html:11]: are the 2 links above to custom fonts not on webstorm?

Comment (abulyousuf) [reply index.html:11]: First link is for favicon icon and next three links for Google Fonts. I am sorry, I should put comments there.

File: css/styles.css (hunk 2 above)
Comment (abulyousuf) [reply css/styles.css:90]: Thank you, Aaron.

**Comment by Gillickd** [comment]
Hi Abul, 
Code is awesome .. and I like the way you used flex end on the images ! 
Great job ... cheers Dave

**Comment by abulyousuf** [comment]
> Hi Abul, Code is awesome .. and I like the way you used flex end on the images ! Great job ... cheers Dave

Thank you, Dave.

**Pull request merged**